jax/experimental/README.md

```diff
@@ -7,6 +7,10 @@ hundred lines of code, so take a look inside and adapt them as you need!
 
 ### Neural-net building with Stax
 
+#### 👉 **Note**: These mini-libraries are meant to be an _inspiration_, but not a prescription. 
+
+To serve that purpose, it is best to keep their code samples minimal; so we generally *will not merge PRs* adding new features. Instead, please send your lovely pull requests and design ideas to more fully-featured libraries like [Haiku](https://github.com/deepmind/dm-haiku), [FLAX](https://github.com/google/flax), or [Trax](https://github.com/google/trax).
```

Comment:
Looks like the official capitalization is "Flax" not "FLAX" (even though on our end we use "JAX" not "Jax"). Tweak the capitalization?

Reply:
Done! :)